src/pipeline_v3/tests/integration/test_search_integration.py

```
@@ -83,6 +102,73 @@
+            # First register the document in the registry
+            registry = search_components["index_manager"].registry
+            doc_id = registry.register_document(
+                source=doc["metadata"]["source"],
+                content_hash=f"hash_{i}",
+                size=len(doc["content"]),
+                modified_time=1640995200,  # Fixed timestamp for testing
+                metadata=doc["metadata"],
+            )
+
+            # Now add to keyword index only
+            from core.registry import IndexType
```

Pylint (3.3.7)

[error] Unable to import 'core.registry'

(E0401)